### DISCUSSION TITLE
updating firedrake

### DISCUSSION THREAD

**Discussion by TahaniSaleem** [firedrakeproject/firedrake#3010] [Firedrake support]
Hello, 

I use Docker images for Firedrake.  I have updated Firedrake to the latest version, but the containers do not run.  

I appreciate your help in fixing that.

**Comment by connorjward** [comment]
Please can you share more information about the error that you are seeing. What are the commands that you are running? What is the error message in full?

**Comment by TahaniSaleem** [comment]
thank you for offering your help. I did not get an error.  

I pulled the image using ( docker pull firedrakeproject/firedrake ). 

I ran it then I tried to run the container but not work and no action happened. 

I tried other images (not Firedrake) and it ran.

**Reply by connorjward** [reply]
When I run with a Docker container I usually run the command `docker run -it <myimage>`. What happens if you do this?
